android/app/build.gradle

```diff
@@ -270,7 +274,7 @@ dependencies {
     implementation("io.mosip:tuvali:0.5.1")
     implementation("io.mosip:inji-vci-client:0.2.0-SNAPSHOT")
     implementation("com.google.code.gson:gson:2.10.1")
-    implementation("io.mosip:vcverifier-aar:1.2.0-SNAPSHOT")
+    implementation("io.mosip:vcverifier-local:3.0.0")
```

Comment:
don't commit local dependencies as it will cause compile time error

Reply:
Updated with 1.1.0 snapshot version